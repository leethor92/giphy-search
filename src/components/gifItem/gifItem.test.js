import React from 'react';
import { shallow, mount } from 'enzyme';
import GifItem from './index';
describe('GifItem', () => {
    it('should render correctly in "debug" mode', () => {
        let gif = {
            id: 1
            };

        const component = shallow(<GifItem gif={gif}/>);

        expect(component).toMatchSnapshot();
    });
});