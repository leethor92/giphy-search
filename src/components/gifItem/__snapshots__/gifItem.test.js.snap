// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`GifItem should render correctly in "debug" mode 1`] = `
<div
  className="col-sm-3"
>
  <div
    className="card"
  >
    <img
      alt={1}
      className="card-img-tag center "
      src={1}
    />
  </div>
</div>
`;
